documentation/changelog/2022-04-13-v011-release.md

```diff
@@ -0,0 +1,57 @@
+---
+tags:
+  - releases
+posted_on_: 2022-04-13T00:00:00Z
+slug: latest-release
```

Comment:
should probably make this v011 so we dont have to change it like the other

Reply:
fixed